ballerina/utils.bal

```diff
@@ -38,7 +39,15 @@ isolated function flattenRecord(record {} r) returns record {} {
     return returnRecord;
 }
 
+isolated function isKnownRecordType(record {} r) returns boolean {
+    if r is time:Civil || 
+       r is time:TimeOfDay || 
+       r is time:Date {
```

Comment:
what if we have other types like `uuid:Uuid` order than the time module? any type coming from a different module.

Reply:
Seems the only known record types we support in `sql:Value` is the time module types.
https://github.com/ballerina-platform/module-ballerina-sql/blob/master/ballerina/types.bal#L36